src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -158,10 +211,12 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
       measurementKeys: [],
     });
 
+    const isUpdatingWidget = typeof onUpdateWidget === 'function' && !!previousWidget;
+
     return (
       <React.Fragment>
         <Header closeButton onHide={closeModal}>
-          {t('Add Widget')}
+          {t(isUpdatingWidget ? 'Edit Widget' : 'Add Widget')}
```

Comment:
I think for the translation parser to work this needs to be:
```suggestion
          {isUpdatingWidget ? t('Edit Widget') : t('Add Widget')}
```